starter-code/src/app/services/access-control-log.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AccessControlLogService {
+  logMessages: Array<{person: string, message: string, createdAt: Date}>;
+  
+
+  constructor() { }
+
+  addAccessItem(person, message){
+    
+    console.log('We are adding sthg');
```

Comment:
What about pushing an object into the array (this.logMessages) instead of doing a console.log?